tests/console/yast2_apparmor.pm

```diff
@@ -24,11 +24,10 @@ sub run {
     assert_script_run("/usr/bin/zypper -n -q in yast2-apparmor");
 
     # start apparmor configuration
-    script_run("yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
-
+    # script_run("yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
```

Comment:
Instead of leaving this line commented, I would leave a comment explaining why this change is necessary.